lib/pos_blockchain.go

```diff
@@ -134,8 +135,48 @@ func (bc *Blockchain) validateDeSoBlockPoS(desoBlock *MsgDeSoBlock) error {
 // that the timestamp is valid, that the version of the header is valid,
 // and other general integrity checks (such as not malformed).
 func (bc *Blockchain) validateBlockGeneral(desoBlock *MsgDeSoBlock) error {
```

Comment:
Maybe rename this `validateBlockIntegrity`? If I understand correctly, this function is intended to validate that hte block is proper formed. Seems like this name suggestion could be more representative of what it does.

This isn't a strong opinion fwiw